specs/sql/Encoding.md

```diff
@@ -0,0 +1,15 @@
+# Encoding
+
+As mentioned in the section on [Statement Types](#statement-types), the core Tableland SQL parser accepts a semicolon-separated list of statements, which are then parsed and evaluated according this Tableland SQL Specification. Internally, the statements are represented using an [abstract syntax tree](https://en.wikipedia.org/wiki/Abstract_syntax_tree) (AST). The internal representation of the nodes of this AST are outside the bounds of the Tableland SQL Specification, however, further details can be found in the [Go Tableland SQL Parser](https://github.com/tablelandnetwork/go-sqlparser) reference implementation.
```

Comment:
"which are then parsed and evaluated according this Tableland SQL Specification", maybe should be "which are then parsed and evaluated according to Tableland SQL Specification"? (`this` -> `to`)

Reply:
"AST are outside" -> "AST is outside" ?